scripts/f5-config.py

```diff
@@ -51,13 +77,13 @@
     r' defaults-from http destination *:8775 recv "200 OK" send "HEAD /'
     r' HTTP/1.1\r\nHost: rpc\r\n\r\n" }',
     r'create ltm monitor http /' + PART + '/' + PREFIX_NAME + '_MON_HTTP_HORIZON { defaults-from http'
-    r' destination *:80 recv "302 Found" send "HEAD / HTTP/1.1\r\nHost:'
+    r' destination *:80 recv "200 OK" send "HEAD / HTTP/1.1\r\nHost:'
```

Comment:
In Juno, I think it gave 302 on Horizon when doing the health check (The send portion) in response. In later versions it returned a 200. Best give it a try now and see what it responds with the send string.